server/controllers/ticket/comment.php

```diff
@@ -25,8 +25,6 @@
  * @apiUse NO_PERMISSION
  * @apiUse INVALID_CONTENT
  * @apiUse INVALID_TICKET
- * @apiUse INVALID_TOKEN
- * @apiUse INVALID_FILE
```

Comment:
same here for INVALID_FILE

Reply:
I guess it's used implicitly by the `FileUploader`? In that case (if this path can return that error) then let's leave it.